### DISCUSSION TITLE
Welcome to instabrute.github.io Discussions!

### DISCUSSION THREAD

**Discussion by dark-player** [dark-player/instabrute.github.io#18] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by dark-player** [comment]
Since we had provided all the instructions and guide to run this project on YouTube, the video was removed from YouTube because of violating their terms and conditions. If you need any help feel free to ask here

**Reply by AyaHa09** [reply]
Hi , i really need help . I am new to all this and I really need to use it i-ll-gally . But I won't hurt anyone. Please help ?

**Comment by 0xj0n1** [comment]
Pretty funny that the IG link is "for _some_ reason not available"
